flang/lib/Lower/OpenACC.cpp

```diff
@@ -2047,7 +2053,7 @@ genACC(Fortran::lower::AbstractConverter &converter,
       std::get<std::optional<Fortran::parser::DoConstruct>>(loopConstruct.t);
   auto loopOp = createLoopOp(converter, currentLocation, semanticsContext,
                              stmtCtx, *outerDoConstruct, eval, accClauseList,
-                             needEarlyExitHandling);
+                             {}, needEarlyExitHandling);
```

Comment:
```suggestion
                             /*combinedConstructs=*/{}, needEarlyExitHandling);
```
Or you can put a default value with the argument